react/src/pages/datasets/DatasetTable.tsx

```diff
@@ -103,11 +103,15 @@ export default function DatasetTable() {
                     }}
                     onUpdate={(dataset_id: string, newDataset: { [key: string]: any }) => {
                         setDatasets(
-                            datasets.map(dataset =>
-                                dataset.dataset_id === dataset_id
-                                    ? { ...dataset, ...newDataset }
-                                    : dataset
-                            )
+                            datasets.map(dataset => {
+                                if (dataset.dataset_id === dataset_id) {
+                                    const updatedDataset = { ...dataset, ...newDataset };
+                                    setInfoDataset(updatedDataset);
```

Comment:
If I'm understanding right, this is because the dialog edits call onUpdate and this state is passed as a prop all the way down?

Reply:
Yep. The 'info dataset' has to be updated or else the contents of the dialog will not update post-edit.